bpf/lib/common.h

```diff
@@ -245,9 +249,23 @@ struct endpoint_info {
 	__u32		pad[4];
 };
 
-struct egress_info {
-	__u32 egress_ip;
-	__u32 tunnel_endpoint;
+struct egress_policy_key {
+	struct bpf_lpm_trie_key lpm_key;
+	__be32 saddr;
+	__be32 daddr;
+};
+
+struct egress_policy {
+	/* Size is the number of IPs set in the gateway_ips field (i.e. the number of
+	 * gateways configured for the policy).
+	 */
+	__u32 size;
+	__be32 egress_ip;
+	__be32 gateway_ips[EGRESS_MAX_GATEWAY_NODES];
+};
+
+struct egress_ct {
```

Comment:
nit, but this can help readability a lot for those unfamiliar with the intent of the code: 
```suggestion
struct egress_ct_entry {
```